app/realtime.php

```diff
@@ -130,25 +200,74 @@
         return $request;
     });
 
-    App::setResource('response', function () {
-        return null;
-    });
+    App::setResource('consoleDB', function () use (&$register) {
+        $consoleDB = new Database();
+        $consoleDB->setAdapter(new MySQLAdapter($register)); // TODO: Add Redis
```

Comment:
@eldadfux Adding Redis here throws errors because of Swoole CoRoutine blocking the socket 😃 

We might want to implement a Redis Connection Pool https://www.swoole.co.uk/docs/modules/redis-connection-pool